File: LeetCode/Simulation/JS/Leetcode1822-sign-of-the-product-of-an-array.js
var arraySign = function (nums) {
  let [nag, n] = [0, nums.length];
  for (let i = 0; i < n; i++) {
    if (nums[i] == 0) return 0;
    if (nums[i] < 0) nag++;
  }
  return nag % 2 ? -1 : 1;
};

File: LeetCode/Simulation/Golang/Leetcode1822-sign-of-the-product-of-an-array.go
package sbgo

func arraySign(nums []int) int {
	nag, n := 0, len(nums)
	for i := 0; i < n; i++ {
		if nums[i] == 0 {
			return 0
		}
		if nums[i] < 0 {
			nag++
		}
	}
	if nag%2 == 1 {
		return -1
	}
	return 1
}
